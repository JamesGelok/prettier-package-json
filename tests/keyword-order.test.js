const { format } = require('../src');

test('It orders keywords in alphabetical order', () => {
  const json = {
    keywords: ['c', 'a', 'b']
  };

  expect(format(json)).toMatchSnapshot();
});
