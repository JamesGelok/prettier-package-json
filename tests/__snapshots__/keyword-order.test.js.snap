// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It orders keywords in alphabetical order 1`] = `
"{
  \\"keywords\\": [
    \\"a\\",
    \\"b\\",
    \\"c\\"
  ]
}"
`;
